doc/glm/first_level_model.rst

```diff
@@ -0,0 +1,221 @@
+
+  contrast_matrix = np.eye(design_matrix.shape[1])
+  basic_contrasts = dict([(column, contrast_matrix[i])
+                for i, column in enumerate(design_matrix.columns)])
+
+Using basic_contrasts, we can construct more interesting contrasts::
+
+  contrasts = {
+    'faces-scrambled': basic_contrasts['faces'] - basic_contrasts['scrambled'],
+    'scrambled-faces': -basic_contrasts['faces'] + basic_contrasts['scrambled'],
+    'effects_of_interest': np.vstack((basic_contrasts['faces'],
+                                      basic_contrasts['scrambled']))
+  }
+
+And compute the contrasts as follows::
+
+  for contrast_id, contrast_val in contrasts.items():
+    z_map = fmri_glm.compute_contrast(
+        contrast_val, output_type='z_score')
+
+.. image:: ../auto_examples/04_glm_first_level_models/images/sphx_glr_plot_spm_multimodal_faces_001.png
+     :target: ../auto_examples/04_glm_first_level_models/plot_spm_multimodal_faces.html#sphx-glr-auto-examples-04-glm-first-level-models-plot-spm-multimodal-faces-py
+     :scale: 60
+
+.. image:: ../auto_examples/04_glm_first_level_models/images/sphx_glr_plot_spm_multimodal_faces_002.png
+    :target: ../auto_examples/04_glm_first_level_models/plot_spm_multimodal_faces.html#sphx-glr-auto-examples-04-glm-first-level-models-plot-spm-multimodal-faces-py
+    :scale: 60
+
+.. image:: ../auto_examples/04_glm_first_level_models/images/sphx_glr_plot_spm_multimodal_faces_003.png
+     :target: ../auto_examples/04_glm_first_level_models/plot_spm_multimodal_faces.html#sphx-glr-auto-examples-04-glm-first-level-models-plot-spm-multimodal-faces-py
+     :scale: 60
+
+
+For full examples on fitting a first level model, look at the following examples:
+
+ref:`sphx-glr-auto-examples-04-glm-first-level-models-plot-spm-multimodal-faces.py`
+
+ref:`sphx-glr-auto-examples-04-glm-first-level-models-plot-fiac-analysis.py`
+
+
+
+Extracting predicted time series and residuals
+==============================================
+
+One way to assess the quality of the fit is to compare the observed and predicted time series of voxels.
+Nilearn makes the predicted time series easily accessible via a parameter called
+`predicted` that is part of the :class:`nilearn.stats.first_level_model.FirstLevelModel`. This parameter
+is populated the when FistLevelModel is initialized with the `minimize_memory` flag set to `False`. ::
+
+  observed_timeseries = masker.fit_transform(fmri_img)
+  predicted_timeseries = masker.fit_transform(fmri_glm.predicted[0])
+
+Here, masker is an object of :class:`nilearn.input_data.NiftiSpheresMasker`. In the figure below,
+predicted (red) and observed (not red) timecourses of 6 voxels are shown.
+
+  .. image:: ../auto_examples/04_glm_first_level_models/images/sphx_glr_plot_predictions_residuals_002.png
+     :target: ../auto_examples/04_glm_first_level_models/plot_predictions_residuals.html#sphx-glr-auto-examples-04-glm-first-level-models-plot-predictions-residuals-py
+     :scale: 60
```

Comment:
This image is really ugly... the is probbaly a problem with the formatting here. Can you improve that ?

Reply:
Yeah, I noticed. There was a scale factor set that I've removed. Hopefully it'll look better now.